docker-compose.yml

```diff
@@ -17,12 +17,14 @@ services:
 
   app:
     # Uncomment this line to use the official program_intent_engagement base image
-    image: openedx/program_intent_engagement
-
+    # image: openedx/program_intent_engagement
+    build:
+      context: .
+      dockerfile: Dockerfile
     container_name: program_intent_engagement.app
     volumes:
-      - .:/edx/app/program_intent_engagement/
-    command: bash -c 'while true; do python /edx/app/program_intent_engagement/manage.py runserver 0.0.0.0:18781; sleep 2; done'
+      - .:/edx/app/program-intent-engagement
```

Comment:
the mount point/working directory was inconsistently using dashes and underscores so the container actually ended up with 2 versions of the code - one at `/edx/app/program-intent-engagement` and one at `/edx/app/program_intent_engagement` the former running the server and the later having the local code mounted.

https://github.com/edx/program-intent-engagement/blob/jnagro/docker-compose-tweaks/0/Dockerfile#L51

it is confused a bit by the repository also has its code in a library folder `program_intent_engagement`. 

An alternative approach might be to make _everything_ use underscores - including the github repository name.